lib/usd/utils/MergePrims.cpp

```diff
@@ -159,6 +161,92 @@ template <class T> auto makeFuncWithContext(const MergeContext& ctx, T&& func)
         func, ctx, ph::_1, ph::_2, ph::_3, ph::_4, ph::_5, ph::_6, ph::_7, ph::_8, ph::_9);
 }
 
+//----------------------------------------------------------------------------------------------------------------------
+// Special transform attributes handling.
+//
+// The goal is to detect that a prim transform has *not* changed even though its representation
+// might have changed. For example it might use different attributes (RotateX vs RotateXYZ) or
+// different transform operations order. (These are in the xformOpOrder attribute.)
+//----------------------------------------------------------------------------------------------------------------------
+
+bool isTransformProperty(const UsdProperty& prop)
+{
+    static const TfToken xformOpOrderToken("xformOpOrder");
```

Comment:
UsdGeomXformOp::IsXformOp() does not include the xformOpOrder token, so we need to check for that too. (Otherwise we'd not write any xformOp because we'd detect they were equivalent but would then change the order which would mess things up!)